release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -62,7 +62,7 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-developer-perspective"]
 ==== Developer Perspective
 
-* With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
+* With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page.
```

Comment:
```suggestion
* With this update, you can export your application in the ZIP file format to another project or cluster by using the *Export application* option on the *+Add* page.
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage

Reply:
acked